doc/specs/#2557 - Settings Keybinding.md

```diff
@@ -0,0 +1,120 @@
+---
+author: Carlos Zamora @carlos-zamora
+created on: 2020-05-14
+last updated: 2020-05-14
+issue id: #2557
+---
+
+# Open Settings Keybinding
+
+## Abstract
+
+This spec outlines an expansion to the existing `openSettings` keybinding.
+
+## Inspiration
+
+As a Settings UI becomes more of a reality, the behavior of this keybinding will be expanded on to better interact with the UI. Prior to a Settings UI, there was only one concept of the modifiable user settings: settings.json.
+
+Once the Settings UI is created, we can expect users to want to access the following scenarios:
+- Settings UI: globals page
+- Settings UI: profiles page
+- Settings UI: color schemes page
+- Settings UI: keybindings page

```

Comment:
_these are provided as non-comprehensive examples of pages that might be in a future settings UI. The rest of the doc assumes these are the pages in the settings UI_

Probably best to make some comment like this